README.md

```diff
@@ -172,6 +172,23 @@ To uninstall, execute:
 kubectl delete -f https://raw.githubusercontent.com/rancher/local-path-provisioner/master/deploy/local-path-storage.yaml
 ```
 
+## Building from source
+
+To build binaries for all supported architectures
+```
+$ CROSS=1 make
+```
+
+To build docker images from those binaries, you will need a local copy of the `manifest-tool` installed.
```

Comment:
I saw this from https://github.com/estesp/manifest-tool:

>UPDATE (Feb 2018): The Docker client PR #138 is merged, providing a docker manifest command that can replace the use of manifest-tool in most scenarios. Other follow-on PRs are in process to add functionality to docker manifest to make it completely functional for all multi-platform image use cases.

Should we use that instead?

Reply:
After this PR is accepted, I would consider it.

Reply:
I would also expect that same new PR to be one that `rancher/dapper` would also be interested in using, so I will open similar PRs together, at that time.